ops/terraform/services/pipeline/modules/bfd_pipeline_dashboard/get_est_or_edt.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -eou pipefail
+
+TZ="America/New_York"
+
+echo "{\"timezone\":\"$(date +%Z)\"}"
```

Comment:
Good idea!

Though, it doesn't seem to work under macos/bsd `date` nor GNU coreutils' `date` without a minor adjustment. Given the state of affairs where _some_ people might be running terraform out-of-band (and potentially living outside of the target TZ 😉), it could be a bit of an unwelcome surprise 😬

I'll leave it up to you, but an explicit `export TZ` or in-lining `TZ` with the call to `date` ought to work, e.g. 
```suggestion
echo "{\"timezone\":\"$(TZ="America/New_York" date +%Z)\"}"
```

Reply:
Ah, I had made the assumption that not inlining `TZ` was functionally equivalent within the context of this script. Guess I let that slip since my timezone _is_ `America/New_York`.

Rather than keep `get_est_or_edt.sh` I've opted to inline the entire script into the `external` `est_or_edt` data source in Terraform. I've also inlined the definition of `TZ` with the call to `date`, as suggested.